contributed_definitions/NXem.nxdl.xml

```diff
@@ -171,145 +236,183 @@
+         * :ref:`NXdata`; at the top-level, conceptually, this is a place for documenting  
            available default plottable data. A default plottable can be useful for  
            research data management systems to show a visual representation of some  
            aspect of the content of the EM session.  
-           It is clear that what constitutes a useful default plot is a matter of  
-           interpretation, somewhat of personal taste, and community standards.  
+           Default plottables not intended to serve every possible analysis and  
+           visualization demand but be a preview. We made this choice because what
+           constitutes a useful default plot is often a matter of interpretation,
+           somewhat of personal taste, and community standards. In effect, default
+           plottables are case- and method-specific.  
          
-         In effect, default plottables are case- and method-specific.
-         Usually a session at a microscope is used to collect multiple signals and
-         images. Examples for possible default plottables could be an arbitrarily
-         taken: secondary, back-scattered, electron image, diffraction pattern,
-         EELS spectra, composition, or orientation mappings to name but a few.
+         Usually a session at a microscope is used to collect multiple signals.
+         Examples for possible default plottables could be arbitrarily taken secondary,
+         back-scattered, electron image, diffraction pattern, EELS spectra, composition,
+         or orientation mappings to name but a few.
          
          **There are a few design choices to consider with sub-ordinate groups:**
          
-         * Above images, spectra, and mappings should be stored as NXdata instances,  
-           ideally formatted in such a way that they can be displayed with  
-           visualization software that can be specific for the file format in which  
-           the data are stored. NeXus specifies only the data model, i.e. the terms  
-           and their relations. These descriptions can be implemented and stored in  
-           JSON, HDF5, XML, or HSDS, file storage, or even other formats, although  
-           HDF5 is the most commonly used.
+         * Images and spectra should be stored as :ref:`NXimage_set` and :ref:`NXspectrum_set`  
+           instances to hold data at the earliest possible step in the computational  
+           processing (which is usually performed with the microscope control and or  
+           integrated analysis software). The formatting of the NXdata groups enables to  
+           display image and spectra with web technology visualization software.  
+           NeXus specifies only the data model, i.e. the terms and their relations.  
+           These descriptions can be implemented and stored in JSON, HDF5, XML, or HSDS,  
+           file storage, or even other formats, although HDF5 is often used.  
```

Comment:
What descriptions?  This is unclear to me.  If this refers to NeXus specifications, then I don't think we need to talk about HDF5 or these other formats here.  Discussion of the implementation of the storage of NeXus data is probably best done elsewhere.

Reply:
removed as indeed maybe better discussed elsewhere